nixos/modules/services/x11/xserver.nix

```diff
@@ -181,6 +181,13 @@ in
         '';
       };
 
+      excludePackages = mkOption {
```

Comment:
I am not convinced moving it to X11 makes sense. Especially since the meaning of “X11 package” is ambiguous – either it is packages from Xorg project (in which case I would not expect it to include e.g. GNOME packages), or it means any graphical package (and then the name does not really make sense, since most GUI apps are based on a toolkit that abstracts the display server, making them no less Wayland than X). And also the latter would imply that the option works for every module under `services.xserver`, which is decidedly not the case – it is only implemented by Xserver and few DEs that tend to install lot of stuff by default.

Reply:
We are not moving any option, just the function is moved away from gnome files.